### PR TITLE
Update GoQuorum to Ethereum client

### PR DESCRIPTION
**Pull request by bgravenorst**
## Pull Request Description
Update GoQuorum to Ethereum client and mention Besu as a supported client.

## Fixed Issue(s)
#48

### CODE REVIEW

File: docs/Concepts/Privacy-manager.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ description: Overview of Tessera privacy manager
 
 # Privacy Manager
 
-Tessera is the Java implementation of GoQuorum's privacy manager.
+Tessera is as the privacy manager for the [GoQuorum](https://docs.goquorum.consensys.net) and
```
Comment (chris-j-h) [inline docs/Concepts/Privacy-manager.md:7]:
```suggestion
Tessera is the privacy manager for the [GoQuorum](https://docs.goquorum.consensys.net) and
```

**Review by chris-j-h — COMMENTED** [review]
Couple of things:

1. *Ethereum client* could be misleading as geth, parity, etc. can't be used with Tessera.  Something like *privacy-enabled Ethereum client* may be more appropriate.  Or alternatively, *Quorum* if that can be considered the root protocol of which GoQuorum and Besu are implementations
2. Tessera has a configurable `Q2T` API.  I think putting something like *Ethereum to Tessera API (formerly Quorum to Tessera API)* will keep consistency between the docs and the config.

File: docs/Concepts/Privacy-manager.md (hunk 1 above)
Comment (bgravenorst) [reply docs/Concepts/Privacy-manager.md]: Done

File: docs/Concepts/TesseraAPI.md [hunk 2]
```diff
@@ -4,15 +4,15 @@ description: Tessera API
 
 # Tessera API
 
-The Tessera API consists of three parts:
+The [Tessera API](https://consensys.github.io/tessera/) consists of the following parts:
 
-* [GoQuorum to Tessera](#goquorum-to-tessera-api)
-* [Peer to peer](#peer-to-peer-api)
-* [Third party](#third-party-api).
+* [`quorum-to-tessera`](#ethereum-to-tessera-api)
```
Comment (chris-j-h) [inline docs/Concepts/TesseraAPI.md:9]:
```suggestion
* [`quorum-to-tessera`](#quorum-to-tessera-api)
```

**Review by NicolasMassart — APPROVED** [review]

**Review by chris-j-h — APPROVED** [review]

**Pull request merged**